.github/workflows/windows.yml

```diff
@@ -142,69 +123,110 @@ jobs:
             -DENABLE_PYTHON=ON `
             -DBUILD_nvidia_plugin=OFF `
             -DCMAKE_DISABLE_FIND_PACKAGE_PkgConfig=ON `
-            -DCUSTOM_OPERATIONS="calculate_grid;complex_mul;fft;grid_sample;sparse_conv;sparse_conv_transpose" `
-            -DOPENVINO_EXTRA_MODULES=${{ env.OPENVINO_CONTRIB_REPO }}\modules `
-            -DCMAKE_BUILD_TYPE=Release `
+            -DCMAKE_BUILD_TYPE=${{ env.CMAKE_BUILD_TYPE }} `
```

Comment:
`CMAKE_BUILD_TYPE` is not required for multi config generators like `Ninja Multi-Config`